### ISSUE TITLE
 how to get the train.h5 and val.h5?

### ISSUE THREAD

**Issue by Andyyoung0507** [xinshi-chen/l2stop#1]
Hello, i am interested in this project. When i reproduce the Sec 5.3, I followed the README.md tutoril. Then when i run the pretrain the model command. The error shows 'OSError 
Unable to open file (unable to open file: name = './train.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)', i found there is no train.h5 and val.h5 in the current directory. So i want to ask how to get these two files.
Thank you!

**Comment by Hanjun-Dai** [comment]
Hi,

Did you download the dataset and put it in corresponding folder?

**Comment by Andyyoung0507** [comment]
Yeah, I download the datasets and unzip the datasets in this folder './dncnn_stop'. There are three folders named 'Set12\Set68\train' in './dncnn_stop/data'.
